src/core/integrators/steepest_descent.cpp

```diff
@@ -100,15 +101,13 @@ bool steepest_descent_step(const ParticleRange &particles) {
 }
 
 void steepest_descent_init(const double f_max, const double gamma,
-                           const int max_steps, const double max_displacement) {
+                           const double max_displacement) {
   params.f_max = f_max;
   params.gamma = gamma;
-  params.max_steps = max_steps;
   params.max_displacement = max_displacement;
 }
 
-int steepest_descent(const int max_steps) {
-  params.max_steps = max_steps;
-  MPI_Bcast(&params, sizeof(SteepestDescentParameters), MPI_BYTE, 0, comm_cart);
-  return integrate(params.max_steps, -1);
+void mpi_bcast_steepest_descent_worker(int, int) {
+  namespace mpi = boost::mpi;
+  mpi::broadcast(comm_cart, params, 0);
```

Comment:
the alias is probably not needed, since you even add more text :D. Why is the return gone?

Reply:
the deleted return belonged to a different function that has been removed